packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -148,6 +150,9 @@ function ConnectorDetails() {
               </div>
             </div>
             <div className={styles.operations}>
+              {data.type === ConnectorType.Email && data.usage !== undefined && (
```

Comment:
should be `data.type !== ConnectorType.Social`. Passwordless connectors can have usage while social can not.